### PR TITLE
[ticket/14596] Deny installation of Ascraeus under PHP 7

### PR DESCRIPTION
**Pull request by 3D-I**
https://tracker.phpbb.com/browse/PHPBB3-14596

PHPBB3-14596

### CODE REVIEW

**Comment by 3D-I** [comment]
It was a PITA, but.. thanks. :)

File: phpBB/language/en/install.php [hunk 1]
```diff
@@ -300,11 +300,11 @@
 	'PHP_REGISTER_GLOBALS'			=> 'PHP setting <var>register_globals</var> is disabled',
 	'PHP_REGISTER_GLOBALS_EXPLAIN'	=> 'phpBB will still run if this setting is enabled, but if possible, it is recommended that register_globals is disabled on your PHP install for security reasons.',
 	'PHP_SAFE_MODE'					=> 'Safe mode',
-	'PHP_SETTINGS'					=> 'PHP version and settings',
-	'PHP_SETTINGS_EXPLAIN'			=> '<strong>Required</strong> - You must be running at least version 5.3.3 of PHP in order to install phpBB. If <var>safe mode</var> is displayed below your PHP installation is running in that mode. This will impose limitations on remote administration and similar features.',
+	'PHP_SETTINGS'					=> 'PHP versions and settings',
+	'PHP_SETTINGS_EXPLAIN'			=> '<strong>Required</strong> - You must be running at least versions 5.3.3 and less than 7 of PHP in order to install phpBB Ascraeus (3.1.x). If <var>safe mode</var> is displayed below your PHP installation is running in that mode. This will impose limitations on remote administration and similar features.',
```
Comment (VSEphpbb) [inline phpBB/language/en/install.php]: Still no good. Should be more like 
>  you  must be running at least version 5.3.3 of PHP (PHP 7 is not supported)) in order...,

And lose the as reads addition. It does not mean anything to end users and this change only applies to this version/branch of phpbb.

**Comment by 3D-I** [comment]
@VSEphpbb
Ok, I will do it. Thanks.

**Comment by 3D-I** [comment]
Somebody please can confirm I did a right commit?
I mean, the procedure was

1 - edited again that file into my local fork
2 - committed and pushed to my remote fork

Travis automaticallt started then (here)

**Comment by Skouat** [comment]
The title of the two last commits are not correct.

**Comment by 3D-I** [comment]
@Skouat 
I aborted the first of the lasts two commits because of this, the very last commit should do though.

**Comment by Skouat** [comment]
Nop!

The chars `[` missing on the two last title ;)

**Comment by 3D-I** [comment]
@Skouat
Thanks. :)

**Comment by 3D-I** [comment]
I do not understand, maybe a travis issue?
Somebody pls can help?

[https://travis-ci.org/phpbb/phpbb/jobs/125133112](url)

**Comment by Skouat** [comment]
It fails because the two commits have a improper title.
You must undo all commits submitted april 22, and resubmit it with no errors

**Comment by DavidIQ** [comment]
^ what he said

**Comment by 3D-I** [comment]
Ok, I thought about that.

With TortoiseGit what command shoul I use then? Or git bash.
Thx.

**Comment by 3D-I** [comment]
I am working on it.. hopefully.
An hint shouldchelp though. :)

**Comment by 3D-I** [comment]
Closing in order to test something.

**Comment by VSEphpbb** [comment]
Please make it stop 😖

**Pull request closed without merging**